### DISCUSSION TITLE
百度网盘svip该如何配置本地磁盘挂载

### DISCUSSION THREAD

**Discussion by ColaHikari** [AlistGo/alist#6193] [General]
![image](https://github.com/alist-org/alist/assets/123285105/9ce5ec87-f886-43f2-869d-4d7a7636a4a2)
![image](https://github.com/alist-org/alist/assets/123285105/22660e6b-62bb-4f7d-84ce-5c73d8c7a026)
采用此配置项，无法实现复制粘贴，无法在线观看，人麻了。

**Comment by ColaHikari** [comment]
将策略改成本地代理仍出现问题

**Comment by ColaHikari** [comment]
还是说通过挂载的方式无法实现在线观看
如果使用非官方接口，以及netdisk ua视频会产生卡顿，我的带宽应该是足够的

**Comment by Phone-IDE** [comment]
我是下载 观看都不行..

**Comment by tediorelee** [comment]
我这里用本地代理可以下载，观看没有尝试还
